tests/unit/event_callback.c

```diff
@@ -0,0 +1,95 @@
+/* vim:expandtab:shiftwidth=2:tabstop=2:smarttab:
+ *
+ * Drizzle Client & Protocol Library
+ *
+ * Copyright (C) 2016 Andreas Bok Andersen (andreas.bok@sociomantic.com)
```

Comment:
Is this the correct copyright notice to have here, or should it be our standard one?